cpp/basix/lagrange.h

```diff
@@ -6,25 +6,29 @@
 
 #include "cell.h"
 #include "finite-element.h"
+#include "lattice.h"
 
 namespace basix
 {
 /// Create a Lagrange element on cell with given degree
-/// @param celltype interval, triangle, quadrilateral, tetrahedral, or
+/// @param[in] celltype interval, triangle, quadrilateral, tetrahedral, or
```

Comment:
Add brief description of what `celltype` is - details of `lattice::type` will be documented where `lattice::type` is declared.